### PR TITLE
Add Remove from team/Leave team

### PR DESCRIPTION
**Pull request by lfbrock**

### CODE REVIEW

**Comment by it33** [comment]
@lfbrock adding proposed tweaks as a PR: https://github.com/mattermost/docs/pull/264

File: source/help/getting-started/managing-members.md [hunk 1]
```diff
@@ -45,14 +45,33 @@ The System Admin is typically a member of the IT staff and has all the privilege
...
+
+## Removing a User from a Team or Leaving a Team
+
+#### Remove from Team
+
+Any Team Administrator has the ability to remove a user from a team by going to the three dot **Main Menu > Manage Members**, and selecting **Remove From Team** in the dropdown menu beside a user entry. 
+
+When a user is removed from a team, if they currently have the team open they will be sent back to the team selection page. The team will no longer show up in their teams list. 
+
+Removing a user from the team does not make the account Inactive. The user will still be able to sign in to the site, and join other teams. They will also be able to rejoin the team they were removed from if they receive another invite, or if the team is an ["Open"](http://docs.mattermost.com/help/settings/team-settings.html#allow-anyone-to-join-this-team) team. If the user does rejoin the team, they will no longer belong to the channels they were previously a part of.
```
Comment (it33) [inline source/help/getting-started/managing-members.md]: The `allow-anyone-to-join-this-team` is being renamed, in my PR I propose removing the link

File: source/help/getting-started/managing-members.md [hunk 2]
```diff
@@ -45,14 +45,33 @@ The System Admin is typically a member of the IT staff and has all the privilege
...
 - Ability to promote and demote other users to and from the System Admin role
 - Ability to make a user account Inactive and to reactivate an Inactive account
- 
 
 #### Inactive 
 
-A System Administrator has the ability to make a user account Inactive by going to **System Console** > **Teams**, and selecting a team to manage. From their they can go to the **User** page for the team, and manage a user's role. 
+A System Administrator has the ability to make a user account Inactive by going to **System Console > Teams**, and selecting a team to manage. From there they can go to the **User** page for the team, and manage a user's role. 
 
 When "Inactive" is selected, the user will no longer be able to log into the system.
```
Comment (it33) [inline source/help/getting-started/managing-members.md]: Is an inactive user also removed from teams and private channels? My PR assumes yes, but I could be wrong.

**Comment by esethna** [comment]
+1

**Comment by jasonblais** [comment]
+1

**Pull request merged**